configure.ac

```diff
@@ -76,5 +76,8 @@ AX_VALGRIND_CHECK
 
 AC_CHECK_HEADER([uthash.h], [], [AC_MSG_ERROR([Unable to find uthash header])])
 
+AM_CFLAGS="-Wall -Wextra -Wmissing-format-attribute -Wmissing-noreturn -Wpointer-arith -Wshadow -Wstrict-prototypes -Wundef -Wunused -Wwrite-strings"
```

Comment:
It looks to me like these aren't actually being used in the build because they are being overridden by the AM_CFLAGS settings in Makefile.am